packages/utils/src/async/parallel.ts

```diff
@@ -1,9 +1,3 @@
-type WorkerType<Return> = Promise<{
-  resultIndex: number
-  workerIndex: number
-  value: Return | Error
-}>
-
 /**
  * Decorates the call of the worker function to return the
```

Comment:
The docs don't match the function implementation anymore.

Reply:
changed